docs/en/00_Getting_Started/00_Server_Requirements.md

```diff
@@ -8,35 +8,30 @@ Our web-based [PHP installer](installation/) can check if you meet the requireme
 
 ## Web server software requirements
 
- * PHP 5.6 and PHP 7.x
- * Once PHP versions become [unsupported by the PHP Project](http://php.net/supported-versions.php),
-   we drop support for those versions in the [next minor release](/contributing/release-process). This means that PHP 5.6 support may be dropped in a 4.x minor release after December 2018.
+ * PHP 7.1+
+   * Once PHP versions become [unsupported by the PHP Project](http://php.net/supported-versions.php),
+     we drop support for those versions in the [next minor release](/contributing/release-process). This means that PHP 7.1 support may be dropped in a 4.x minor release after December 2019.
  * We recommend using a PHP accelerator or opcode cache, such as [xcache](http://xcache.lighttpd.net/) or [WinCache](http://www.iis.net/download/wincacheforphp).
```

Comment:
Can we remove this doc about opcode cache since PHP has had this bundled since 5.5?

Reply:
Yep, good idea, done.